packages/react-composites/src/composites/CallWithChatComposite/SidePane.tsx

```diff
@@ -84,6 +85,9 @@ export const EmbeddedPeoplePane = (props: {
   callAdapter: CallAdapter;
   chatAdapter: ChatAdapter;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onChatButtonClick: () => void;
```

Comment:
*Not for this PR*. We should really split off `EmbeddedPeoplePane` and `EmbeddedChatPane` to their own files.

Reply:
Yep definitely necessary but will do in separate PR.